packages/react-instantsearch-dom-maps/package.json

```diff
@@ -34,8 +34,8 @@
     "clean": "rm -rf dist",
     "watch": "yarn build:cjs --watch",
     "build": "yarn build:cjs && yarn build:es && yarn build:umd",
-    "build:cjs": "babel src --out-dir dist/cjs --ignore __tests__,__mocks__ --quiet",
-    "build:es": "BABEL_ENV=es babel src --out-dir dist/es --ignore __tests__,__mocks__ --quiet",
+    "build:cjs": "babel src --root-mode upward --out-dir dist/cjs --ignore '**/__tests__/**/*.js','**/__mocks__/**/*.js' --quiet",
+    "build:es": "BABEL_ENV=es babel src --root-mode upward --out-dir dist/es --ignore '**/__tests__/**/*.js','**/__mocks__/**/*.js' --quiet",
     "build:umd": "BABEL_ENV=rollup rollup -c rollup.config.js",
```

Comment:
`BABEL_ENV` was already named like that but why `rollup` instead of `umd`?

`rollup` seems to be a detail, we want to detect if it's a UMD bundle.

Reply:
It doesn't matter to me, we can rename it.